site-src/concepts/cluster-profile-api.md

```diff
@@ -0,0 +1,57 @@
+# ClusterProfile API Overview
+
+This document provides an overview of the ClusterProfile (fka ClusterInventory) API.

```

Comment:
```suggestion
This document provides an overview of the ClusterProfile API.
```

We don't need to mention the formally know as part here. ClusterInventory API on a conceptual level still exists and the sig repo is cluster-inventory-api.